src/modules/site/templates/_includes/macros.html

```diff
@@ -77,12 +77,16 @@
   </div> {# /col-md-2 #}
   <div class="col-md-6">
     {% for action in actions %}
-      <a 
-        class="{{ action.get('classes', '') }}"
-        style="{{ action.get('style','') }}"
-        href="{{ url_for(action.get('url')) }}"
-      > {{ action.get('label') }}
-      </a>
+      {% if not action.get('admin_only', False) or (action.get('admin_only', False) and session['is_admin']) %}

```

Comment:
you can use session.get('is_admin') which won't throw a KeyError, and return None instead if the key does not exist.